engine/src/parser/expression_utils.cpp

```diff
@@ -54,6 +54,7 @@ bool is_unary_operator(operator_type op) {
 	case operator_type::BLZ_CHAR_LENGTH:
 	case operator_type::BLZ_STR_LOWER:
 	case operator_type::BLZ_STR_UPPER:
+        case operator_type::BLZ_STR_INITCAP:
```

Comment:
please  fix spacing so that it matches the rest

Reply:
addressed